src/TorchSharp/NN/Linear.cs

```diff
@@ -13,48 +12,53 @@ namespace Modules
     {
         public sealed class Linear : torch.nn.Module<Tensor, Tensor>
         {
-            internal Linear(IntPtr handle, IntPtr boxedHandle) : base(handle, boxedHandle)
+            internal Linear(long inputSize, long outputSize, bool hasBias = true, Device device = null, ScalarType? dtype = null) : base(nameof(Linear))
```

Comment:
Is there any reason to not have this constructor to be `public`? It does not take handles anymore.

Reply:
Only because the Python-likeness we're striving for would prefer that users use the factories in `torch.nn` instead of the constructors.